packages/react-components/react-popover/library/src/components/Popover/Popover.types.ts

```diff
@@ -22,8 +22,8 @@ export type PopoverProps = Pick<PortalProps, 'mountNode'> & {
   appearance?: 'brand' | 'inverted';
 
   /**
-   * Can contain two children including {@link PopoverTrigger} and {@link PopoverSurface}.
-   * Alternatively can only contain {@link PopoverSurface} if using a custom `target`.
+   * Can contain two children including PopoverTrigger and PopoverSurface.
+   * Alternatively can only contain PopoverSurface if using a custom `target`.
```

Comment:
_nit:_ You could in its place use backticks to make the text look different

```suggestion
   * Can contain two children including `PopoverTrigger` and `PopoverSurface`.
   * Alternatively can only contain `PopoverSurface` if using a custom `target`.
```